demo/index.html

```diff
@@ -1,29 +1,39 @@
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
-  <title>vaadin-radio-button Examples</title>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"
+      />
```

Comment:
Please move this closing bracket to previous line.

Reply:
Done.